load_constants.py

```
@@ -59,3 +73,89 @@ def reload_prompts():
+    }
+
+def load_system_prompts():
+    paths = update_paths()
+    try:
+        with open(paths['SYSTEM_PROMPT_FILE'], 'r', encoding="utf-8") as f:
+            SYSTEM_PROMPT = f.read()
+        with open(paths['JOURNAL_SYSTEM_PROMPT_FILE'], 'r', encoding="utf-8") as f:
+            JOURNAL_SYSTEM_PROMPT = f.read()
+        return SYSTEM_PROMPT, JOURNAL_SYSTEM_PROMPT
+    except FileNotFoundError as e:
+        raise Exception(f"Failed to load system prompts: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)